go/programs/bubble_sort.go

```diff
@@ -0,0 +1,51 @@
+package programs
+
+import "fmt"
+
+func BubbleSort(array []int) []int {
```

Comment:
Give some meaning full name to variable not just array

Reply:
done